src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -297,18 +293,18 @@ private String checkLinks()
 	private void reportsAfterAddLinks(CrossReferenceReporter xrefReporter, DuplicateIdentifierReporter duplicateIdentifierReporter, Map<String, Map<String, Integer>> preAddLinksReport) throws SQLException, IOException, Exception
 	{
 		logger.info("Counts of references to external databases currently in the database ({}), AFTER running AddLinks", this.dbAdapter.getConnection().getCatalog());
-		//reporter.printReport();
 		Map<String, Map<String,Integer>> postAddLinksReport = xrefReporter.createReportMap();
-		logger.info("\n"+xrefReporter.printReport(postAddLinksReport));
+		String report = xrefReporter.printReport(postAddLinksReport);
```

Comment:
`printReport` kind of sounds like a method with a void return type that outputs to the screen.  Does the method do something in addition to creating the String that's returned?  If not, maybe `createReport` is a more precise name.